components/statuslight/index.css

```diff
@@ -11,243 +11,262 @@ governing permissions and limitations under the License.
-  --spectrum-statuslight-nonsemantic-gray-color: var(--spectrum-gray-visual-color);
-  --spectrum-statuslight-nonsemantic-red-color: var(--spectrum-red-visual-color);
-  --spectrum-statuslight-nonsemantic-orange-color: var(--spectrum-orange-visual-color);
-  --spectrum-statuslight-nonsemantic-yellow-color: var(--spectrum-yellow-visual-color);
-  --spectrum-statuslight-nonsemantic-chartreuse-color: var(--spectrum-chartreuse-visual-color);
-  --spectrum-statuslight-nonsemantic-celery-color: var(--spectrum-celery-visual-color);
-  --spectrum-statuslight-nonsemantic-green-color: var(--spectrum-green-visual-color);
-  --spectrum-statuslight-nonsemantic-seafoam-color: var(--spectrum-seafoam-visual-color);
-  --spectrum-statuslight-nonsemantic-cyan-color: var(--spectrum-cyan-visual-color);
-  --spectrum-statuslight-nonsemantic-blue-color: var(--spectrum-blue-visual-color);
-  --spectrum-statuslight-nonsemantic-indigo-color: var(--spectrum-indigo-visual-color);
-  --spectrum-statuslight-nonsemantic-purple-color: var(--spectrum-purple-visual-color);
-  --spectrum-statuslight-nonsemantic-fuchsia-color: var(--spectrum-fuchsia-visual-color);
-  --spectrum-statuslight-nonsemantic-magenta-color: var(--spectrum-magenta-visual-color);
+	/* Static tokens */
+	--spectrum-statuslight-corner-radius: 50%;
+	--spectrum-statuslight-border-width: var(--spectrum-border-width-100);
+
+	--spectrum-statuslight-text-to-visual-75: var(--spectrum-status-light-text-to-visual-75);
+	--spectrum-statuslight-text-to-visual-100: var(--spectrum-status-light-text-to-visual-100);
+	--spectrum-statuslight-text-to-visual-200: var(--spectrum-status-light-text-to-visual-200);
+	--spectrum-statuslight-text-to-visual-300: var(--spectrum-status-light-text-to-visual-300);
+
+	/* Size */
+	--spectrum-statuslight-height: var(--spectrum-component-height-100);
+	--spectrum-statuslight-dot-size: var(--spectrum-status-light-dot-size-medium);
+	--spectrum-statuslight-line-height: var(--spectrum-line-height-100);
+	--spectrum-statuslight-line-height-cjk: var(--spectrum-cjk-line-height-100);
+
+	/* Font */
+	--spectrum-statuslight-font-family: var(--spectrum-default-font-family);
+	--spectrum-statuslight-font-weight: var(--spectrum-regular-font-weight);
+	--spectrum-statuslight-font-style: var(--spectrum-default-font-style);
+	--spectrum-statuslight-font-size: var(--spectrum-font-size-100);
+
+	/* Space */
+	--spectrum-statuslight-spacing-dot-to-label: var(--spectrum-statuslight-text-to-visual-100);
+	--spectrum-statuslight-spacing-top-to-dot: var(--spectrum-status-light-top-to-dot-medium);
+	--spectrum-statuslight-spacing-top-to-label: var(--spectrum-component-top-to-text-100);
+	--spectrum-statuslight-spacing-bottom-to-label: var(--spectrum-component-bottom-to-text-100);
+
+	/* Color */
+	--spectrum-statuslight-content-color-default: var(--spectrum-neutral-content-color-default);
+
+	/* TODO: is --spectrum-neutral-subdued-content-color-default supposed to point to gray-600, not gray-700? */
```

Comment:
this comment needs to be removed (the subdued-content-color-default property is pointing to the correct gray-600 token now)